README.md

```diff
@@ -1,4 +1,60 @@
-node-voicemail-admin
-====================
+# Asterisk Voicemail Administrator Application
+
+A command line administration tool for managing mailboxes in Asterisk voicemail
+
+# Installation
+
+```bash
+$ git clone https://github.com/asterisk/node-voicemail-admin.git
+$ cd node-voicemail-admin
+$ npm install -g .
```

Comment:
I would change this to npm install. npm install -g will install all the node-voicemail-admin dependencies globally which will pollute the global space and is not necessary to run the application from the checkout directory.